website/server/controllers/api-v3/user.js

```diff
@@ -223,6 +224,17 @@ api.deleteUser = {
 
     await user.remove();
 
+    if (req.body.feedback) {
+      let mailOptions = {
+        from: '"Habitica" <noreply@habitica.com>',
+        to: 'admin@habitica.com',
+        subject: 'Feedback received from account deletion',
+        html: `<b>Feedback:</b> ${req.body.feedback}`,
```

Comment:
Should we include any information about the user, such as an email address? If not, should we state that the feedback is anonymous?